### PR TITLE
Switched from pem to p12

### PR DESCRIPTION
**Pull request by cjllanwarne**

### CODE REVIEW

**Comment by mcovarr** [comment]
An IDE search for "P12" leads me to believe more things should be changed...

File: src/main/config/cromwell.conf.ctmpl [hunk 1]
```diff
@@ -18,7 +18,7 @@ google {
   authScheme = "service"
 
   serviceAuth {
-    p12File = "{{.Data.service_auth_p12_file}}"
+    pemFile = "{{.Data.service_auth_pem_file}}"
```
Comment (kcibul) [inline src/main/config/cromwell.conf.ctmpl]: @mmonnar is this secret in place in all the environments now?

Comment (cjllanwarne) [inline src/main/config/cromwell.conf.ctmpl]: Plot twist: it was there all along.....

Comment (cjllanwarne) [inline src/main/config/cromwell.conf.ctmpl]: Oh wait, no, the **file** was there all along, this value we added manually. Yes, it's there now.

Comment (cjllanwarne) [inline src/main/config/cromwell.conf.ctmpl]: Oh wait (2): we put it in place for **dev**. I don't know who manages the other environments?

**Comment by mcovarr** [comment]
:+1:

**Comment by geoffjentry** [comment]
:+1:

**Pull request merged**